x-pack/metricbeat/module/openmetrics/module.yml

```diff
@@ -0,0 +1,3 @@
+name: openmetrics
+metricsets:
+- collector
```

Comment:
Happy to see this module implemented as light module, but I think it would be better to implement it as a native module. This way it could be used as an input for other light modules.

Implementation could be by now something like this:
```
import (
     ...
     "github.com/elastic/beats/metricbeat/module/prometheus/collector"
)

func init() {
        mb.Registry.MustAddMetricSet("openmetrics", "collector", collector.New,
                mb.WithHostParser(hostParser),
        )
}
```

Reply:
I'm ok with that! @exekias any thoughts before changing it?

Reply:
+1, also we expect this to change in the future, so a native implementation will make that easier